README.md

```diff
@@ -1,2 +1,30 @@
 # comp110-worksheet-B
-Base repository for COMP110 worksheet B
+
+Fallout 4 hacking mechanic
+
+word = guess
+cword = correct word
+
+Enter terminal
+input attempt
+
+if word contains 0 likeness
```

Comment:
How is likeness calculated? Your task for the worksheet was to provide pseudocode for an algorithm to compute this